### PR TITLE
Week 15 ToDo App

### PR DESCRIPTION
**Pull request by Yennie-Leek**
Link to my deployed project: 
[https://elated-gates-17548d.netlify.app/](url)

My reflections on how this week's project turned out:
It was a fun project. Im happy to practice how to work in styled components more.

Things I'd like to have clarified or explained in more detail:
Styling of checkboxes.. what different alternatives that is out there. Its like a jungle to find a good match and how to implement it my own code.. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+I can add and delete my tasks that I need to keep track of.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
 
 ## The problem
+I have built the project with minor issues, but with some help from my team,
+the lectures, google, SO I solved them.
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I did get stuck on styling the checkboxes and if I had more time I would continue to fix them.
```
Comment (sirimatsdotter) [inline README.md:11]: I used React Custom Checkboxes and I highly recommend it!

File: code/public/index.html [hunk 2]
```diff
@@ -4,6 +4,10 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Shadows+Into+Light&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Mukta:wght@300;500;700&display=swap" rel="stylesheet">
```
Comment (sirimatsdotter) [inline code/public/index.html:10]: Nice font! :)

File: code/src/components/TodosDone.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+const TaskWrapper = styled.section`
+  font-family: 'Shadows Into Light', cursive;
+  font-size: 20px;
+`
+export const TodosDone = () => {
+  const list = useSelector(store => store.todos.items)
+  const listDone = list.filter(item => item.isComplete).length 
+  return (
+    <TaskWrapper>
+      <h3>Tasks done: {listDone}/{list.length}</h3>
```
Comment (sirimatsdotter) [inline code/src/components/TodosDone.js:14]: Nice work!

**Review by sirimatsdotter — APPROVED** [review]
Great job Yennie! 💯 

You nailed all the blue requirements what I can see and your code is super structured and neat - easy to read and follow! 😍 Great work with the styled components and using luxon for date and time formatting! 

I agree custom checkboxes would be the cherry on top, so if you have time you could implement that! Other than that I think it works and looks great! 😃

File: code/src/components/Main.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import { TodoForm } from './TodoForm'
+
+const MainWrapper = styled.main`
+ background-color: #f38d6e;
+ color:#5c5756;
+ height: 100vh;
+ width: 100wh;
+ display: flex;
+ flex-direction: column;
+ align-items:center;
```
Comment (KarolCislowski) [inline code/src/components/Main.js:15]: Indentation is off here 😢

File: code/src/components/TodoForm.js [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+  border-bottom: double #5c5756;
+  color:#363232;
+  font-family: 'Shadows Into Light', cursive;
+  font-size:30px;
+  text-align:center;
+  padding:0px;
+`
+const AddButton = styled.button`
+  background-color:#f9ac45;
+  color:#363232;
```
Comment (KarolCislowski) [inline code/src/components/TodoForm.js:34]: You should configure your Prettier to autoformat your styled components syntax.

**Review by KarolCislowski — APPROVED** [review]
Nice app with a clean interface, good job 🥳

**Pull request closed without merging**